scene/3d/path_3d.cpp

```diff
@@ -185,30 +198,34 @@ void PathFollow3D::_update_transform(bool p_update_xyz_rot) {
 		return;
 	}
 
+	// Actual update.
+	transform_dirty = false;
```

Comment:
It's probably safe to set it to `false` immediately after checking. The rest of the checks here ensure we are not in a state where we care about transforms anyway. And if this changes, then something somewhere should already call `update_transform` to force an update.

Setting `path` and path's curve does that. Baked length is an exception, it seems, but that may actually be a separate problem that also needs solving at some point.

Reply:
fixed.